src/main/java/algorithms/leetcode/easy/SquareRoot.java

```diff
@@ -0,0 +1,40 @@
+package algorithms.leetcode.easy;
+
+/*
+69. Sqrt(x)
+Given a non-negative integer x, return the square root of x rounded down to the nearest integer. The returned integer
+should be non-negative as well. You must not use any built-in exponent function or operator.
+For example, do not use pow(x, 0.5) in c++ or x ** 0.5 in python.
+
+Example 1:
+Input: x = 4
+Output: 2
+Explanation: The square root of 4 is 2, so we return 2.
+
+Example 2:
+Input: x = 8
+Output: 2
+Explanation: The square root of 8 is 2.82842..., and since we round it down to the nearest integer, 2 is returned.
+
+Constraints:
+0 <= x <= 231 - 1
+ */
+
+public class SquareRoot {
+    public static int mySqrt(int x) {
+        long sqrt = 0;
+        if(x == 0 || x == 1) return x;
+        for(long i = 2;i<=x;i++){
```

Comment:
can be refactored

Reply:
Yes... If I change it to x/2, it is working fine. But in leetcode, the time taken is more
1. For approach i<=x, it is showing as 23ms  - https://leetcode.com/problems/sqrtx/submissions/1075172597/
2. For approach i<=x/2, it is showing as 57ms - https://leetcode.com/problems/sqrtx/submissions/1077818341/